docs/source/mayavi/auto/adjust_cropping_extents.py

```diff
@@ -14,9 +14,9 @@
 are modified to propagate them to the filter. For more information
 on creating GUIs with Traits:
 
-    http://code.enthought.com/projects/traits/docs/html/tutorials/traits_ui_scientific_app.html
+    http://docs.enthought.com/traitsui/tutorials/traits_ui_scientific_app.html
 
-    http://code.enthought.com/projects/traits/documentation.php
+    http://docs.enthought.com/traits
```

Comment:
Let's also include 
https://support.enthought.com/hc/en-us/articles/204469620-Introductory-materials-for-Traits-and-Traits-UI